src/test/java/org/rulez/demokracia/pdengine/ObtainBallotTest.java

```diff
@@ -75,6 +75,27 @@ public void obtainBallot_stores_the_ballot() {
 		assertTrue(vote.ballots.contains(ballot));
 	}
 
+	@tested_feature("Manage votes")
+	@tested_operation("Obtain ballot")
+	@tested_behaviour("the number of ballots obtained with adminKey are recorded for \"admin\"")
+	@Test
+	public void obtainBallot_increases_recordedBallots_when_adminKey_is_admin() {
+		String voteId = adminInfo.voteId;
+		Vote vote = voteManager.getVote(voteId);
+		int originalObtainedBallots = vote.getRecordedBallots("admin");
+		String ballot = voteManager.obtainBallot(voteId, "admin");
+		assertEquals(originalObtainedBallots + 1, vote.getRecordedBallots("admin").intValue());
+	}
 
-
+	@tested_feature("Manage votes")
+	@tested_operation("Obtain ballot")
+	@tested_behaviour("the number of ballots obtained with adminKey are recorded for \"admin\"")
```

Comment:
this is another feature in another issue

Reply:
This PR contains these tasks:
#194
#195

Reply:
Ant this is #195, not #194

Reply:
I changed the PR name I found out the problem was that. My first commit contained only the #195 and my second commit contained the #194.
I hope it is not a problem to solve two or more ticket in one PR.
If it is please let me know and I will create separate PRs for the tickets. :)

Reply:
it is okay to solve multiple issues in one PR.